packages/labs/react/src/create-component.ts

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
+const NODE_MODE = false;
```

Comment:
NODE_MODE is always false?

Reply:
`NODE_MODE` flag is flipped to `true` for node builds via rollup.

Reply:
so rollup replaces this with `const NODE_MODE = true`? Cool

Reply:
https://github.com/lit/lit/blob/9f802646d955198cbaf6e521283fe137e7f5b7a6/rollup-common.js#L447